src/actions/MatrixActionCreators.js

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2017 Vector Creations Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {
+    createMatrixActionCreator,
+    createMatrixSyncActionCreator,
+} from './actionCreators';
+
+// Events emitted from the matrixClient that we want to dispatch as actions
+// via MatrixActionCreators. See createMatrixActionCreator.
+const REGISTERED_EVENTS = [
+    "accountData",
+];
+
+export default {
+    actionCreators: [],
+    actionCreatorsStop: [],
+
+    start(matrixClient) {
+        this.actionCreators = REGISTERED_EVENTS.map((eventId) =>
+            createMatrixActionCreator(matrixClient, eventId),
+        );
+        this.actionCreators.push(createMatrixSyncActionCreator(matrixClient));
+
+        this.actionCreatorsStop = this.actionCreators.map((ac) => ac());
```

Comment:
OOB we decided to modify action creators so that they are all of the same structure and are either:
 - a function that takes arguments and returns an Action (JSON dict with `action` member) or 
 - a function that returns a function `(dispatch) => {...}` that, when given the `dispatch` function, will dispatch actions asynchronously.